src/notification.ts

```diff
@@ -127,6 +114,7 @@ export class BaseNotification {
    * [scab]: https://npmjs.com/package/@inrupt/solid-client-authn-browser
    *
    * @param sessionFetch
+   * @deprecated If a custom fetch function is required then it can be set in the constructor
```

Comment:
@ajacksified can we just wholesale deprecate this, given we never really intended it to be used and there's an alternative (pass in via constructor)? The usage of this module is still fairly low, and I don't think we've really taught this API much.

Reply:
Yeah, with this being refactored the method isn't useful anyways. I should have marked that as internal to begin with. Because the likelihood of this being used is low, I think it's safe to remove.